source/gui/settingsDialogs.py

```diff
@@ -1636,31 +1636,53 @@ def makeSettings(self, settingsSizer):
 		settingsSizer.Add(sizer)
 
 		# Translators: The label for the edit field in symbol pronunciation dialog to change the pronunciation of a symbol.
-		changeSizer = wx.StaticBoxSizer(wx.StaticBox(self, wx.ID_ANY, _("Change symbol")), wx.VERTICAL)
+		changeSizer = wx.StaticBoxSizer(wx.StaticBox(self, wx.ID_ANY, _("Change selected symbol")), wx.VERTICAL)
+
+		# Used to ensure that event handlers call Skip(). In the case of handling EVT_KILL_FOCUS, not calling skip can 

```

Comment:
You mention EVT_KILL_FOCUS specifically here, but we no longer use that event in this dialog. I'm concerned this might be confusing on future reflection.